<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textkeypad</class>
 <widget class="QDialog" name="textkeypad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(3, 96, 126);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButOne">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButTwo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But9">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But0">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButBack">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 17pt &quot;Ubuntu&quot;;
image: url(:/image/enterbutton.png);
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButQ">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButW">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButE">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButY">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButR">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButT">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButO">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButU">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButI">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButP">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButK">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButD">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButEnter">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
image: url(:/image/enterbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButUpper_Lower">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>UP/Low</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButA">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButG">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButS">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButL">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButF">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButJ">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButH">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Butcomma">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButC">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButQues">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButZ">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButB">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButX">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButDot">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButV">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButM">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButN">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButBackSpace">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 75 15pt &quot;Ubuntu&quot;;
font:bold;
image: url(:/image/backspacebutton.png);</string>
   </property>
   <property name="text">
    <string>BackSpace</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
